app/indexers/hyrax/file_set_indexer.rb

```diff
@@ -1,57 +1,115 @@
 # frozen_string_literal: true
+
 module Hyrax
-  class FileSetIndexer < ActiveFedora::IndexingService
```

Comment:
i'm worried about this name swap. this would be a high-impact breaking change, right?